Python/Product/PythonTools/PythonTools/OutliningTaggerProvider.cs

```diff
@@ -199,6 +199,14 @@ internal class OutliningTagger : ITagger<IOutliningRegionTag> {
                 int cnt = 0;
                 var text = snapshot.GetText(start, length);
 
+                // The VS Provider is fragile to us splitting in between \r\n.  
+                // Check here that we haven't done that and correct the error.
+                if (text[0] == '\n' && start >= 1 && snapshot.GetText(start - 1, 1) == "\r") {

```

Comment:
But how did we get here in the first place? We must have an off-by-one error somewhere (as shown when you change a file to \n only and now regions are off by one line)